files/settings-default.cfg

```diff
@@ -1,244 +1,709 @@
+# and fog.  For low frame rate screenshots of scenic vistas, this
+# setting should be set high, and viewing distances adjusted
+# accordingly.  This value must be greater than or equal to 1.
+exterior cell load distance = 1
 
 [GUI]
-scaling factor = 1.0
 
-# 1 is fully opaque
+# These two settings determine the background color of the tool tip
+# and the crosshair when hovering over an item owned by an NPC.  The
+# color definitions are composed of four floating point values between
+# 0.0 and 1.0 inclusive, representing the red, green, blue and alpha
+# channels.  The alpha value is currently ignored.  The crosshair
+# color will have no effect if the "crosshair" setting in the HUD
+# section is disabled.  These colors are used only if the "show owned"
+# setting is enabled in the Game section.
+color background owned = 0.15 0.0 0.0 1.0
+color crosshair owned = 1.0 0.15 0.15 1.0
+
+# This boolean setting enables or disables the "red flash" overlay
+# that provides a visual clue when the character has taken damage.
+hit fader = true
+
+# This floating point setting controls the transparency of the GUI
+# windows.  The value should be between 0.0 (transparent) and 1.0
+# (opaque).  The setting can be adjusted in game with the "Menu
+# Transparency" slider in the Prefs panel of the Options menu.
 menu transparency = 0.84
 
-# 0 - instantly, 1 - max. delay
-tooltip delay = 0
+# This floating point setting scales the GUI interface windows.  The
+# value must be greater than 0.0.  A value of 1.0 results in the
+# default scale.  Values much larger than 2.0 may result in user
+# interface components being inaccessible.
+scaling factor = 1.0
+
+# Stretch or shrink the introductory movie, new game screen, and
+# loading screens to fill the specified video resolution.  The default
+# assets have a 4:3 aspect ratio, but other assets may have other
+# resolutions.  If this setting is false, the assets will be centered
+# in their correct aspect ratio.
+stretch menu background = false
 
+# Enable or disable subtitles for NPC spoken dialog (and some sound
+# effects).  Subtitles will appear in a tool tip box in the lower
+# center of the screen.  The setting can be toggled in game with the
+# "Subtitles" button in the Prefs panel of Options menu.
 subtitles = false
 
-hit fader = true
+# Set the delay between when you begin hovering over an item and when
+# it's tooltip appears.  This setting is a floating point value
+# between 0.0, which displays the tool tip instantly and 1.0 which
+# results in the maximum delay (approximately 1.5 seconds).  This
+# setting does not affect the tooltip delay for object under the
+# crosshair in the "look mode", only widgets in the GUI windows.  This
+# setting can be adjusted in game with the "Menu Help Delay" slider in
+# the Prefs panel of the Options menu.
+tooltip delay = 0.0
+
+# Enable or disable the werewolf overlay.  Unable to evaluate fully
+# due to issues with becoming a werewolf.
```

Comment:
You can do ```player->becomewerewolf``` in the console to become a werewolf, but generally this feature won't work in  the Bloodmoon expansion is not installed.

The setting controls a full screen texture effect that is displayed only in first person mode for werewolf players.